blazarclient/utils.py

```diff
@@ -93,7 +93,8 @@ def get_item_properties(item, fields, mixed_case_fields=None, formatters=None):
             else:
                 field_name = field.lower().replace(' ', '_')
             if not hasattr(item, field_name) and isinstance(item, dict):
-                data = item[field_name]
+                # display null if the resource does not have a field
+                data = item.get(field_name, "null")
```

Comment:
We should use the empty string instead, otherwise users may try to reserve something with `property=null` which won't work. By if the user sees an empty string in the table, it should be more clear.